backend/x11/backend.c

```diff
@@ -1,25 +1,29 @@
 #define _POSIX_C_SOURCE 200112L
+
 #include <assert.h>
 #include <limits.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <time.h>
+
+#include <wlr/config.h>
+
+#include <X11/Xlib-xcb.h>
 #include <wayland-server.h>
+#include <xcb/xcb.h>
+#include <xcb/xfixes.h>
+#include <xcb/xinput.h>
+
 #include <wlr/backend/interface.h>
 #include <wlr/backend/x11.h>
-#include <wlr/config.h>
 #include <wlr/interfaces/wlr_input_device.h>
 #include <wlr/interfaces/wlr_keyboard.h>
```

Comment:
What's up with all the weird newlines?

Reply:
I like the way includes are groupped like this, but it's inconsistent with other files in the repo.

Reply:
I just alphasort headers. No need to think too hard about it.

Reply:
You don't, otherwise `#include "thing"` would show up before `#incldue <thing>`.

You create two groups, alphasort each, and don't separate the two with a newline.